app/presenters/licences/view-licence-returns.presenter.js

```diff
@@ -96,8 +98,22 @@ function _status(returnLog) {
   // 'today' would be flagged as overdue when it is still due (just!)
   today.setHours(0, 0, 0, 0)
 
-  if (status === 'due' && dueDate < today) {
-    return 'overdue'
+  if (status === 'due') {
+    if (dueDate < today) {
```

Comment:
If we swap this to be `if (today > dueDate) { ... }` then it's in line with the comparison we make on line 114 and (for me anyway) the logic flow is a little clearer.